### DISCUSSION TITLE
MetaMask issue

### DISCUSSION THREAD

**Discussion by VParth2005** [Cyfrin/foundry-full-course-cu#1248] [Q&A]
MetaMask isnt letting me confirm my secret recovery phrase how can I fix this?

**Comment by ghost** [comment]
You either: (a) wrote the recovery phrase in the incorrect order or (b) one of the words are wrong.

If you cannot validate the phrase, just try again by creating a new profile.

**Comment by Githopmelain77** [comment]
Hello, 
i have an issue for when i first login in MetaMask the value of each coin is seen,but  then it despair, all the coins value except  BNB. i try to contact with metamask suport but they ask me for my email, but i was told to never communicate with email, because Metamask will never contact or send an email. so i was advice to use github. hope i will find a solution for this issue. please help
i appreciate any kind help i could find.

**Comment by heheboii11** [comment]
try to import the contract address of the bnb on the network you are looking for, Ig this will solve the issue.

**Reply by Githopmelain77** [reply]
i really appreciate your response to my concerns. 
i didn’t get what you mean, it is not about  to import, the thing is i have all my portfolio list of coins and the quantities of each coin, but not the value of each coin in USD.
note: i am not good in understanding straight forward explanation, please if its ok can you be more clear and easy to get your point. thank you for your time
